core/src/com/group66/game/helpers/DifficultyManager.java

```diff
@@ -0,0 +1,52 @@
+package com.group66.game.helpers;
+
+
+// TODO: Auto-generated Javadoc
+/**
+ * The Class DifficultyManager.
+ */
+public class DifficultyManager {
+    
+    /** The difficulty. */
+    public static String difficulty;
```

Comment:
I know it makes much more sense to use enums but I'm just not acquainted with it. Will include this in future code refactoring.